### ISSUE TITLE
add warning when some a11y issue can be detected

### ISSUE THREAD

**Issue by ged-odoo** [odoo/owl#53]
# Motivation

Some accessibility issues may be discovered by Owl.  We are not good at accessibility, and it is difficult to test, but it would be simple to give some warning whenever we have a clear case of a missing accessibility attribute.

# Spec

In QWeb, whenever we compile a generic dom node, and only if we are in `dev` mode, call a `_validateAccessibility` method, which would do the following:

- if the node is an image and has no `alt` attribute, generate a warning (unless it has a `aria-labelledby` attribute)
